### PR TITLE
Add BlockManagerInterface

### PR DESCRIPTION
**Pull request by core23**
## Subject

Import interface from `SonataBlockBundle`, because there in only one implementation here.

Refs: https://github.com/sonata-project/SonataBlockBundle/pull/958

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added `Sonata\BlockBundle\Model\BlockManagerInterface`

### Deprecated
- Deprecated `Sonata\BlockBundle\Model\BlockManagerInterface` usage
```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Is this interface needed or cant we just implements both Manager and PageableManagerInterface ? 

Might be also interesting to know why the PageableManagerInterface doesnt extend the ManagerInterface

**Comment by VincentLanglet** [comment]
Since all the getPager method seems deprecated in this project, I think adding a BlockManagerInterface is not needed, it just need to be abandon in favor of the ManagerInterface

**Comment by core23** [comment]
Reworked this PR. The interface will now contain all public methods from the `BlockManager`.

File: src/Model/BlockManagerInterface.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     */
+    public function updatePosition($id, $position, $parentId = null, $pageId = null, $partial = true);
+
+    /**
+     * @param int $page
+     * @param int $limit
+     *
+     * @return PagerInterface
+     */
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = []);
```
Comment (VincentLanglet) [inline src/Model/BlockManagerInterface.php]: This method need to be mark as deprecated/to be removed in next major
https://github.com/sonata-project/SonataPageBundle/blob/5bd4b5cde9833b25ce15e323a9fc5a7b69d18d3d/src/Entity/BlockManager.php#L73-L78

Comment (core23) [reply src/Model/BlockManagerInterface.php]: Removed that method

**Review by VincentLanglet — DISMISSED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thanks @core23

**Pull request merged**